src/twisted/newsfragments/11913.misc

```diff
@@ -0,0 +1 @@
+Drop support for Python 3.7.
```

Comment:
Maybe this file should be `.removal` and maybe it should also mention that the twisted[contextvar] extra was also removed.

Thanks